test/utils/update/mocks.ts

```diff
@@ -286,7 +287,8 @@ export const FETCH_INDIVIDUAL_TRUSTEE_DATA_MOCK: IndividualTrusteeData[] = [
       poBox: "",
       premises: "100",
       postalCode: "56335",
-      region: "Sierra Madre"
+      region: "Sierra Madre",
+      is_service_address_same_as_usual_residential_address: "No"
```

Comment:
Is there an explicit test line or test we can add to cover the yes/no scenarios?

Reply:
Any reason why some key names in the same interface are `camelCased` and others are `snake_cased`? I'd imagine consistent casing would be preferred?

Reply:
done